Sources/Framework/Views/AnswerView.swift

```diff
@@ -98,7 +93,7 @@ class AnswerView: UIView, AnswerViewProviding {
         scrollView.edgeAnchors == edgeAnchors
 
         verticalStack.edgeAnchors == scrollView.edgeAnchors + stackInsets
-        verticalStack.widthAnchor == (widthAnchor - stackInsets.left - stackInsets.right) ~ .custom(999)
+        verticalStack.widthAnchor == (widthAnchor - stackInsets.left - stackInsets.right) ~ 999
```

Comment:
Any constraint priority below 1000 is optional. IMO If you dont have many constraints you need to prioritise ~low / ~high is better readable

Reply:
Yes, I am aware of that but in this particular case not using this exact constraint priority caused a lot problems with Autolayout.